src/components/buttonPanel.js

```diff
@@ -1,37 +1,42 @@
+import PropTypes from 'prop-types';
 import Button from './button';
 
-const ButtonPanel = () => (
+const ButtonPanel = ({ clickHandler }) => (
   <div className="button-container">
     <div className="top-buttons">
-      <Button name="AC" property="basic-op" />
-      <Button name="+/-" property="basic-op" />
-      <Button name="%" property="basic-op" />
-      <Button name="/" property="basic-op" />
+      <Button name="AC" property="basic-op" clickHandler={clickHandler} />
```

Comment:
- [ ] Optional - Consider changing the component filename from `buttonPanel.js` to `ButtonPanel.js`